### DISCUSSION TITLE
Escaping literal {{

### DISCUSSION THREAD

**Discussion by Dave-McCraw** [handlebars-lang/handlebars.js#1786] [Q&A]
I'm having this issue in Wiremock, but I think it's my understanding of Handlebars that is at fault. Any pointers gratefully received.

I have something that looks like https://jsfiddle.net/65yotkvb/ where I need the literal `{{` to be preserved.

Based on https://handlebarsjs.com/guide/expressions.html#escaping-handlebars-expressions I tried the following two approaches: 

https://jsfiddle.net/3264c7jq/
https://jsfiddle.net/0n4t9d6r/

As you can see, neither of these is correct. I'd be happy to PR an improvement to the docs if anyone can help me figure out what I'm doing wrong.

Thanks!

D.

**Comment by jaylinski** [comment]
This seems to work, as described in the docs:
```
<script src="https://unpkg.com/handlebars@latest/dist/handlebars.js"></script>
<!-- 
If you need a specific version:
<script src="https://unpkg.com/handlebars@4.3.0/dist/handlebars.js"></script> -->

<div id="output"></div>
<script type="template/handlebars" id="template">encodedField="abcd\{{1234"</script>
```

Does this solve your issue?
